x/cellarfees/types/params.go

```diff
@@ -14,16 +25,109 @@ func ParamKeyTable() paramtypes.KeyTable {
 
 // DefaultParams returns default cellarfees parameters
 func DefaultParams() Params {
-	return Params{}
+
+	return Params{
+		// Rough number of blocks in 2 weeks, or ~2 fee accrual cycles for one cellar
+		AuctionBlockDelay: 201600,
+		// Rough number of blocks in 28 days, the time it takes to unbond
+		RewardEmissionPeriod: 403200,
+		// Initial rate at which an auction should decrease the price of the relevant coin from it's starting price
+		InitialPriceDecreaseRate: sdk.NewDec(347000000000000),
+		// Blocks between each auction price decrease
+		PriceDecreaseBlockInterval: 10,
+	}
 }
 
 // ParamSetPairs returns the parameter set pairs.
 func (p *Params) ParamSetPairs() paramtypes.ParamSetPairs {
-	return paramtypes.ParamSetPairs{}
+	return paramtypes.ParamSetPairs{
+		paramtypes.NewParamSetPair(KeyAuctionBlockDelay, &p.AuctionBlockDelay, validateAuctionBlockDelay),
+		paramtypes.NewParamSetPair(KeyRewardEmissionPeriod, &p.RewardEmissionPeriod, validateRewardEmissionPeriod),
+		paramtypes.NewParamSetPair(KeyInitialPriceDecreaseRate, &p.InitialPriceDecreaseRate, validateInitialPriceDecreaseRate),
+		paramtypes.NewParamSetPair(KeyPriceDecreaseBlockInterval, &p.PriceDecreaseBlockInterval, validatePriceDecreaseBlockInterval),
+	}
 }
 
 // ValidateBasic performs basic validation on cellarfees parameters.
 func (p *Params) ValidateBasic() error {
+	if err := validateAuctionBlockDelay(p.AuctionBlockDelay); err != nil {
+		return err
+	}
+	if err := validateRewardEmissionPeriod(p.RewardEmissionPeriod); err != nil {
+		return err
+	}
+	if err := validateInitialPriceDecreaseRate(p.InitialPriceDecreaseRate); err != nil {
+		return err
+	}
+	if err := validatePriceDecreaseBlockInterval(p.PriceDecreaseBlockInterval); err != nil {
+		return err
+	}
+	return nil
+}
+
+func validateAuctionBlockDelay(i interface{}) error {
+	blockDelay, ok := i.(uint64)
+	if !ok {
+		return fmt.Errorf("invalid parameter type: %T", i)
```

Comment:
Nit: Think it would be cleaner to replace these fmt.Errorf's with errors from an errors.go file, e.g. https://github.com/PeggyJV/sommelier/blob/main/x/cork/types/errors.go

and later formatted with extra info e.g. 
https://github.com/PeggyJV/sommelier/blob/main/x/auction/keeper/keeper.go#L169